frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/StaticEmbedSetupPane.tsx

```diff
@@ -174,48 +183,90 @@ export const StaticEmbedSetupPane = ({
+          >{t`Parameters`}</Tabs.Tab>
+          <Tabs.Tab
+            value={TABS.Appearance}
+            onClick={() => setActiveTab(TABS.Appearance)}
+          >{t`Appearance`}</Tabs.Tab>
         </Tabs.List>
-        <Tabs.Panel value={TABS.Overview}>
-          <OverviewSettings
-            resourceType={resourceType}
-            selectedServerCodeOption={selectedServerCodeOption}
-            serverEmbedCodeSlot={getServerEmbedCodePane(TABS.Overview)}
-          />
-        </Tabs.Panel>
-        <Tabs.Panel value={TABS.Parameters}>
-          <ParametersSettings
-            activePane={activePane}
-            resourceType={resourceType}
-            resourceParameters={resourceParameters}
-            embeddingParams={embeddingParams}
-            lockedParameters={lockedParameters}
-            parameterValues={parameterValues}
-            iframeUrl={iframeUrl}
-            displayOptions={displayOptions}
-            serverEmbedCodeSlot={getServerEmbedCodePane(TABS.Parameters)}
-            onChangeEmbeddingParameters={setEmbeddingParams}
-            onChangeParameterValue={(id: string, value: string) =>
-              setParameterValues(state => ({
-                ...state,
-                [id]: value,
-              }))
-            }
-            onChangePane={setActivePane}
-          />
-        </Tabs.Panel>
-        <Tabs.Panel value={TABS.Appearance}>
-          <AppearanceSettings
-            activePane={activePane}
-            resourceType={resourceType}
-            iframeUrl={iframeUrl}
-            displayOptions={displayOptions}
-            serverEmbedCodeSlot={getServerEmbedCodePane(TABS.Appearance)}
-            onChangePane={setActivePane}
-            onChangeDisplayOptions={setDisplayOptions}
-          />
+        <Tabs.Panel value={activeTab}>
+          {activeTab === TABS.Overview ? (
+            <OverviewSettings
+              resourceType={resourceType}
+              selectedServerCodeOption={selectedServerCodeOption}
+              serverEmbedCodeSlot={getServerEmbedCodePane(TABS.Overview)}
+            />
+          ) : activeTab === TABS.Parameters ? (
+            <SettingsTabLayout
+              settingsSlot={
+                <ParametersSettings
+                  resourceType={resourceType}
+                  resourceParameters={resourceParameters}
+                  embeddingParams={embeddingParams}
+                  lockedParameters={lockedParameters}
+                  parameterValues={parameterValues}
+                  onChangeEmbeddingParameters={setEmbeddingParams}
+                  onChangeParameterValue={(id: string, value: string) =>
+                    setParameterValues(state => ({
+                      ...state,
+                      [id]: value,
+                    }))
+                  }
+                />
+              }
+              previewSlot={
+                <>
+                  <PreviewModeSelector
```

Comment:
This is moved from both `ParametersSettings` and `AppearanceSettings`, the structures are similar that's why React doesn't unmount the preview when changing between these 2 tabs.